Sources/AWSSDKSwiftCore/Message/AWSResponse+HAL.swift

```diff
@@ -15,30 +15,28 @@
 import Foundation
```

Comment:
Specific import. I know hasn't come up in this pr, but maybe we should do it anyway.